screwdriver/screwdriver.go

```diff
@@ -41,11 +42,39 @@ type Step struct {
 	Command string `json:"command"`
 }
 
+// PairSlice is an associative array which preserves order
+type PairSlice struct {
+	Body []struct {
+		Key   string
+		Value string
+	}
+}
+
+// UnmarshalJSON replaces JSON of a normal associative array to PairSlice
+func (PS *PairSlice) UnmarshalJSON(data []byte) error {
+	var output []struct {
+		Key   string
+		Value string
+	}
+	r := regexp.MustCompile(`"(.*?)" *: *"(.*?)"`)
```

Comment:
`regexp.MustCompile` should be global for safe and to decrease initializing cost.

https://golang.org/pkg/regexp/#MustCompile

Reply:
I fixed it.